apps/angular/function-call-effect/src/app/app.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { ActionsComponent } from './action.component';
+import { UserService } from './user.service';
+
+@Component({
+  standalone: true,
+  imports: [FormsModule, ActionsComponent],
+  selector: 'app-root',
+  template: `
+    <nav class="flex w-full items-center border border-b">
+      Profil used:
+      <form class="m-4 w-48">
+        <select
+          [(ngModel)]="userService.name"
+          [ngModelOptions]="{ standalone: true }"
```

Comment:
Do you really need `standalone: true` ?  This makes it easier to display errors in the template since you won't have to use `controls['name']` ?